packages/components/src/TemporaryAddressChange/Component.purs

```diff
@@ -161,14 +164,25 @@ render self@{ state: { startDate, endDate, streetAddress, zipCode, countryCode,
         , label: "Börjar från"
         }
 
+    isIndefiniteCheckbox =
+      InputCheckbox.inputCheckbox
+        { type_: InputCheckbox.Checkbox
+        , name: "indefinite"
+        , value: Nothing
+        , checked: false
```

Comment:
Should this be `state.isIndefinite`? Always setting it `false` implies that whenever the state changes (and this thing is re-rendered), the checkbox gets unchecked.

Reply:
Oh, you don't actually use it for anything in `InputCheckbox`. Or am I missing something?

Reply:
:thinking: